src/main/java/com/Timos/TicTacToeInputProducer/Controller/TicTacToeInputController.java

```diff
@@ -0,0 +1,21 @@
+package com.Timos.TicTacToeInputProducer.Controller;
+
+import com.Timos.TicTacToeInputProducer.Service.TicTacToeInputService;
+import com.Timos.TicTacToeInputProducer.model.InputMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TicTacToeInputController {
+
+    @Autowired
+    private TicTacToeInputService service;
+
+    @PostMapping(value = "/TicTacToeInput", consumes = "application/json")
```

Comment:
It's a HTTP endpoint, `input` is a given and overly generic. And it's the TicTacToe application so also implicit. Given the ultimate goal for the application `/move` might be a sensible choice